commons/src/main/java/com/powsybl/commons/config/PlatformConfig.java

```diff
@@ -83,19 +88,21 @@ private static ModuleConfigRepository loadModuleRepository(Path[] configDirs, St
      * Reads from yaml file if it exists, else from xml file, else from properties file.
      */
     private static ModuleConfigRepository loadModuleRepository(Path configDir, String configName) {
-        Path yamlConfigFile = configDir.resolve(configName + ".yml");
+        Path yamlConfigFile = configDir.resolve(configName + ".yaml");
+        Path ymlConfigFile = configDir.resolve(configName + ".yml");
+        Path xmlConfigFile = configDir.resolve(configName + ".xml");
```

Comment:
I think we should add a warning when multiple files exist. Here's what docker-compose does :
```
$ docker-compose up
WARNING: Found multiple config files with supported names: docker-compose.yml, docker-compose.yaml
WARNING: Using docker-compose.yml
```
Initially I was thinking of only doing that for config.yaml and config.yml, but probably it should be generalized to all ModuleConfigRepositories. For PropertiesModuleConfigRepository, we don't know the filenames beforehand but maybe using *.properties is enough and doesn't give false positives.
What do you think ?

Reply:
Yes, I agree. I will add this modification.